doc/concepts/pipeline-concepts/datum/glob-pattern.md

```diff
@@ -0,0 +1,124 @@
+# Glob Pattern
+
+A glob pattern defines datums in your Pachyderm repository and how
+your data processing is spread among Pachyderm worker pods. A standalone
+datum can be a whole
+repository, each top filesystem object in the root folder,
+each filesystem object in a subdirectory, and so on. Each time
+Pachyderm detects new commits in your repository, it starts the
+pipeline and processes the whole datum. Therefore, there might be
+a significant difference between processing the whole repository
+or one directory in the repository. You need to consider this
+behavior, when you design your pipeline.
+
+Defining how your data is spread among workers is one of
+the most important aspects of distributed computation and is
+the fundamental idea around concepts, such as Map and Reduce.
+
+Instead of confining users to data-distribution patterns,
+such as Map — split everything as much as possible, — and
```

Comment:
Dashes? Would commas be better? Map, where you split everything as much as possible, and Reduce, where all data must be grouped, Pachyderm....